### PR TITLE
fix reverse proxy to https

### PR DESCRIPTION
**Pull request by darkonie**
a fix for https reverse proxy

### CODE REVIEW

**Comment by malnick** [comment]
:ship:

**Comment by orsenthil** [comment]
@mesosphere-mergebot test-it

**Comment by orsenthil** [comment]
@mesosphere-mergebot test-it again now, yeah, that one existed previously.

File: api/pull.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ type findMastersInExhibitor struct {
 
 func (f *findMastersInExhibitor) findMesosMasters() (nodes []Node, err error) {
 	if f.getFn == nil {
-		return nodes, errors.New("Could not initialize HTTP GET function. Make sure you set getFn in the constructor.")
+		return nodes, errors.New("could not initialize HTTP GET function. Make sure you set getFn in constractor")
```
Comment (orsenthil) [inline api/pull.go]: Minor spell fix. It should be `constructor`.

**Comment by orsenthil** [comment]
@darkonie , you might also want to rebase your changes on top of master and push this PR again.

I see this error in the back.

`stderr: 'Failed to merge in the changes.'`

**Pull request merged**